include/swift/Parse/Lexer.h

```diff
@@ -461,7 +461,11 @@ class Lexer {
   };
 
 private:
-  /// nul character meaning kind
+  /// Nul character meaning kind.
+  /// There are 3 kinds of meaning about nul character in Lexer.
+  /// 1. BufferEnd is string buffer terminator.
+  /// 2. Embedded is embedded nul character.
+  /// 3. CodeCompletion is code completion marker.
   enum class NulCharacterKind { BufferEnd, Embedded, CodeCompletion };
```

Comment:
Please make this:
```cpp
/// Nul character meaning kind.
enum class NulCharacterKind {
  /// String buffer terminator.
  BufferEnd,
  /// Embedded is embedded nul character.
  Embedded,
...
```
numbered list (1,2,3) might be confusable with step-by-step procedure.

Reply:
Okey, I will fix it.